src/reducers/toolbarOverlay.js

```diff
@@ -0,0 +1,11 @@
+export const showOverlay = (state, { id }) => ({
+  toolbarOverlay: {
+    shortcutId: id
+  }
+})
```

Comment:
Why not just use `toolbarOverlay: id`? Why the extra object wrapper?

Reply:
will do.